packages/bbui/src/bbui.css

```diff
@@ -45,6 +45,11 @@
   --purple: #806fde;
   --purple-dark: #130080;
 
+  --error-bg: rgba(226, 109, 105, 0.3);
```

Comment:
This is slightly confusing, as it is not the background of the error. This is the error icon, right?

Reply:
No this is actually a background in this case, there is an error background for when there's a literal JS error - I just gave it a specific colour that fits inline with the icon colours